src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator {
```

Comment:
```suggestion
public class Circle extends Figure {
```
якщо ти імплументуєш інтерфейс на рівні батьківського інтерфейсу то це перенесеться усім нащадкам